packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerTests/FWFWebViewHostApiTests.m

```diff
@@ -465,6 +465,8 @@ - (void)testContentInsetsSumAlwaysZeroAfterSetFrame {
   XCTAssertTrue(CGRectEqualToRect(webView.frame, CGRectMake(0, 0, 300, 100)));
 }
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130300 || __IPHONE_OS_VERSION_MAX_ALLOWED >= 160400 || \
+    __TV_OS_VERSION_MAX_ALLOWED >= 160400
```

Comment:
We shouldn't need this check on tests, since most clients aren't running our native tests, and it's fine for us to require a new Xcode for development.